odoo/fields.py

```diff
@@ -1771,12 +1772,16 @@ def convert_to_column(self, value, record, values=None, validate=True):
             return None
         # Detect if the binary content is an SVG for restricting its upload
         # only to system users.
-        if value[:1] == b'P':  # Fast detection of first 6 bits of '<' (0x3C)
-            decoded_value = base64.b64decode(value)
-            # Full mimetype detection
-            if (guess_mimetype(decoded_value).startswith('image/svg') and
-                    not record.env.user._is_system()):
-                raise UserError(_("Only admins can upload SVG files."))
+        if value[:2] in [b'PH', b'PD']:
+            # if fast detection of first 12 bits of '<?' or '<s'
+            try:
+                decoded_value = base64.b64decode(value, validate=True)
```

Comment:
It seems to me the `test_90_binary_svg` test fails on runbot because you added here the `validate=True`, and then here it's produced the `binascii.Error` that goes in the `except` and does nothing, so it doesn't raise the UserError 🤔

Reply:
Yes, I'm investigating that